src/ProcessManagerBundle/Tool/Report.php

```diff
@@ -0,0 +1,235 @@
+<?php
+/**
+ * @date    17/11/2017 11:53
+ * @author  Wojciech Peisert <wpeisert@divante.pl>
+ */
+
+namespace ProcessManagerBundle\Tool;
+use ProcessManagerBundle\Exception\NonExistentReportFileException;
+
+/**
+ * Class Report
+ *
+ * Prepares import process report from log file
+ *
+ * @package ProcessManagerBundle\Tool
+ */
+class Report
+{
+    const SKIP_LINES_SHORTER_THAN = 50;
+
+    const EVENT_TOTAL = 'import_definition.total:';
```

Comment:
Logging doesn't happen here, ImportDefinitions should use a Service from ProcessManager for Logging